sanic/compat.py

```diff
@@ -10,6 +11,7 @@
 from multidict import CIMultiDict  # type: ignore
 
 from sanic.helpers import Default
+from sanic.log import error_logger
```

Comment:
This is causing a circular import issue. We _could_ go the option of a larger refactor. However, given the nature of this PR and the context in which it is called, I think we should opt for the shortcut and just put this import inside the `pypy_os_module_patch` function. This seems find enough for me. If you want to tackle a more elegant way, LMK.

Reply:
Now there's a new error I'm trying to run the examples\blueprints.py

```
RuntimeError:
        An attempt has been made to start a new process before the
        current process has finished its bootstrapping phase.

        This probably means that you are not using fork to start your
        child processes and you have forgotten to use the proper idiom
        in the main module:

            if __name__ == '__main__':
                freeze_support()
                ...

        The "freeze_support()" line can be omitted if the program
        is not going to be frozen to produce an executable.
```

To get it to work I added

```
if __name__ == '__main__':
    from multiprocessing import freeze_support
    freeze_support()

    app.run(host="0.0.0.0", port=9999, debug=True)
```

I think there's a problems with forking

Reply:
also the printing on terminal is a mess

![Screenshot 2023-04-17 060827](https://user-images.githubusercontent.com/17798647/232368687-990ecd84-aebe-4073-95be-66581efe1cc1.png)

Reply:
a workaround is force motd to be False on pypy + windows. maybe ?

Reply:
Pypy probably needs to switch their terminal handling to use wchar i.e. UTF-16 output instead of 8 bit char that causes the problem you are seeing. This is something they really need to fix, and I don't see much point in Sanic (and all other Unicode-enabled Python software out there) working around the issue.

Reply:
@Tronic Fixed 🥳 Sanic with PyPy ready to run on all platforms

![Screenshot 2023-04-17 222146](https://user-images.githubusercontent.com/17798647/232590045-f33c9dfb-9183-464e-b83a-9a9129e3af12.png)